ocs_ci/utility/workloadfixture.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf8 -*-
+
+"""
+This module contains functions implementing functionality of workload
+fixtures in ocs-ci.
+
+.. moduleauthor:: Filip Balák
+"""
+
+
+import json
+import logging
+import os
+import threading
+import time
+
+from ocs_ci.utility.prometheus import PrometheusAPI
+
+
+logger = logging.getLogger(__name__)
+
+
+def measure_operation(
```

Comment:
we already have such function in https://github.com/red-hat-storage/ocs-ci/blob/d7597383e8c4ecbb09c6c432d135abb7eb9be25b/tests/manage/monitoring/conftest.py#L28

Reply:
It is moved from `ocs-ci/tests/manage/monitoring/conftest.py` to `ocs_ci/utility/workloadfixture.py' in this PR. This will replace the original `measure_operation` function.